### ISSUE TITLE
Adding Min and Max to a number input breaks label placement.

### ISSUE THREAD

**Issue by SerbianLastName** [antoniandre/wave-ui#105]
```
<w-input class="mb2" type="number" min="0" max="10" v-model="storeReportForm.blazyHypeMeter"
            label="Blazy Hype Number" outline :validators="[settings.validators.required]" />
```
Results in the label being placed much higher than it should be.

![2023-03-16 (3)](https://user-images.githubusercontent.com/76258792/225756185-f7dd4ebf-3394-49d2-bee2-2d2c5bf0c0e5.png)

```
<w-input class="mb2" type="number" max="10" v-model="storeReportForm.blazyHypeMeter" label="Blazy Hype Number"
            outline :validators="[settings.validators.required]" />
```

Using only min, or only max, leaves the label where it should be.

![2023-03-16 (2)](https://user-images.githubusercontent.com/76258792/225756277-a363f34c-67a9-4f7c-841a-d8739549ac44.png)

Using WaveUI with Vue3, can provide any additional info as needed.

**Comment by antoniandre** [comment]
Hi @SerbianLastName, 
I couldn't reproduce this exactly, but I think the problem will be solved anyway with my change in v3.1.
Hope this helps, let me know :)

**Comment by antoniandre** [comment]
Closing due to inactivity. Please reopen if not solved.
